pdf2archive

```diff
@@ -75,6 +75,8 @@ OPTIONS:
                       specified via the command line options.
   --validate          Validate  the resulting file. The  validation is done with
                       VeraPDF, you need a working Java installation.
+  --validate-only     Perform only the validation on the input file, again using
+  		      VeraPDF
```

Comment:
Please use spaces instead of 'tab' here; on GitHub, this line is not aligned.

Reply:
Done!